api/src/pcapi/routes/backoffice_v3/finance/blueprint.py

```diff
@@ -561,3 +562,127 @@ def comment_incident(finance_incident_id: int) -> utils.BackofficeResponse:
             "backoffice_v3_web.finance_incident.get_incident", finance_incident_id=incident.id, active_tab="history"
         )
     )
+
+
+@finance_incident_blueprint.route("/incident/<int:finance_incident_id>/validate", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def get_finance_incident_validation_form(finance_incident_id: int) -> utils.BackofficeResponse:
+    finance_incident = finance_models.FinanceIncident.query.options(
+        sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents).load_only(
+            finance_models.BookingFinanceIncident.newTotalAmount
+        ),
+        sa.orm.joinedload(finance_models.FinanceIncident.venue)
+        .load_only(offerer_models.Venue.name)
+        .joinedload(offerer_models.Venue.reimbursement_point_links)
+        .joinedload(offerer_models.VenueReimbursementPointLink.venue)
+        .load_only(offerer_models.Venue.name),
+    ).get_or_404(finance_incident_id)
```

Comment:
```suggestion
    finance_incident = finance_models.FinanceIncident.query.filter_by(id=finance_incident_id).options(
        sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents).load_only(
            finance_models.BookingFinanceIncident.newTotalAmount
        ),
        sa.orm.joinedload(finance_models.FinanceIncident.venue)
        .load_only(offerer_models.Venue.name)
        .joinedload(offerer_models.Venue.reimbursement_point_links)
        .joinedload(offerer_models.VenueReimbursementPointLink.venue)
        .load_only(offerer_models.Venue.name),
    ).one_or_none()

    if not finance_incident:
        return NotFound()
```

Il semble que `get_or_404()` ne prend pas les jointures, donc ça règlera ton problème de requêtes en trop.
J'ai testé sur ta branche :)

Reply:
Ah top ! je savais pas merci